### ISSUE TITLE
Edit colors for table in dark mode

### ISSUE THREAD

**Issue by katz95** [spotify/backstage#742]
Would love if someone could help us adjust some colors for the table in dark mode!

The header that says "Column One" should be white (#FFFFFF). The rest of the headers should be #EEEEEE. 

The darker rows are too dark at this moment. They should be #333333. 

Thank you!!

<img width="874" alt="Screenshot 2020-05-07 at 6 59 26 PM" src="https://user-images.githubusercontent.com/61153904/81323089-272cdd80-9095-11ea-8cac-94e09d80c5f2.png">
